index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1" />
     <meta name="description" content="Analyse the financial markets using beautiful visualisations and unique trading tools, powered by a fast and intuitive interface." />
-    <link rel="stylesheet" type="text/css" href="./dist/smartcharts.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="./dist/smartcharts.css?c089caa34345a4b55bd5ba3aa32378de" media="screen" />
```

Comment:
Pls check the diff before you request for review. Pls. Don't let me repeat this again. Maybe you would want to update the npm script to not overwrite this file, but it takes effort and no one took it up yet.

Reply:
@brucebinary oops I don't really know how I miss this one